helm/blog-app/templates/namespace.yaml

```
@@ -0,0 +1,16 @@
+{{- if .Values.namespace.create }}
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: {{ .Values.namespace.name }}
+  labels:
+    {{- with .Values.namespace.labels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)